packages/drivers/odsp-driver/src/odspDocumentStorageManager.ts

```diff
@@ -689,6 +690,7 @@ export class OdspDocumentStorageService implements IDocumentStorageService {
                 overalltime: overallTime,
                 networktime: networkTime,
                 clienttime: clientTime,
+                msEdge: msEdge,
```

Comment:
Pls add another column here - isAfd set to true or false, based on whether the header is set or not. We might remove the headers col if we dont need it for cost, but this other column will be present.

Reply:
Added